tf-build-tools/tf-test/src/test/groovy/com/deliveredtechnologies/terraform/fluent/api/TfStateTest.groovy

```diff
@@ -0,0 +1,16 @@
+package com.deliveredtechnologies.terraform.fluent.api
+
+
+import spock.lang.Shared
+import spock.lang.Specification
+
+class TfStateTest extends Specification {
+
+
+    @Shared
+    TfState tfState
```

Comment:
I like @Shared. I think it is also possible to initialize it here, rather than setupSpec()

Reply:
yea. good catch